packages/eslint-plugin/README.md

```diff
@@ -0,0 +1,72 @@
+# eslint-plugin-mirrorful
+
+This directory contains a plugin containing ESLint rules for Mirrorful. This plugin should only be installed into a pre-existing [Mirrorful](https://github.com/mirrorful/mirrorful) project.
+
+See it in action [here](/examples/tailwind-next/).
+
+## Installation
+
+You'll first need to install [ESLint](https://eslint.org/):
+
+```sh
+npm i eslint --save-dev
+```
+
+Next, install `eslint-plugin-mirrorful`:
+
+```sh
+npm install eslint-plugin-mirrorful --save-dev
+```
+
+## Usage
+
+Add `mirrorful` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+
+```json
+{
+    "plugins": [
+        "mirrorful"
+    ]
+}
+```
+
+
+Then, simply extend the recommended configuration.
+
+```json
+{
+    "extends": [
+        "plugin:mirrorful/recommended",
+    ]
+}
+```
+
+Alternatively, you can individually configure the rules you want to use under the rules section.
+
+```json
+{
+    "rules": {
+        "mirrorful/no-hardcoded-colors": 2
+    }
+}
+```
+
```

Comment:
```suggestion
# Development
```

Reply:
Can we move # Development and Publishing to the bottom? Also, this is a silly ask: but can you add some emojis and a screenshot to this README?